ethereum/networks/src/main/java/tech/pegasys/teku/networks/Eth2NetworkConfiguration.java

```diff
@@ -254,6 +254,10 @@ public void spec(Spec spec) {
     }
 
     public Eth2NetworkConfiguration build() {
+      return build(true);
+    }
+
+    public Eth2NetworkConfiguration build(final boolean forBeaconNode) {
```

Comment:
i wrote the other comment below when i was looking for this, but basically, have 
`Eth2NetworkConfigurationBuilder.forBeaconNode(true/false)` and default it to whatever makes sense, then build can continue to have no parameter.

Reply:
yeah I passed it explicitly in some cases to clarify it